src/client/internal.ts

```diff
@@ -243,6 +243,16 @@ export interface ChannelReceiveContextPayload {
     context: Context;
 }
 
+export interface OpenFinChannelConnectionEvent {
```

Comment:
Deeeeefinitely nothing in `@types/openfin` that just gives us this type, or at least something we can extend?

Reply:
Same comment as for the notifications equivalent:

"Cool. For now, would move into connection.ts, unexported since anywhere else using this is probably doing something wrong. And consider extending WindowEvent<,>, which will give you the last 4 fields here"

Reply:
Switched to using a signal to fire this event as its more reliable and doesn't get lost in the runtime when there is multiple windows trying to connect at the same time.

Reply:
Cool. Then this can be deleted?